Dockerfile

```diff
@@ -1,15 +1,17 @@
 FROM golang:1.21-alpine AS build
```

Comment:
Changes on this file must be reflected on GitHub Workflows. At least, in the `release.yml`, I think.